k8s-plugin/src/main/kotlin/com/amazon/spinnaker/keel/k8s/artifactSupplier/GitRepoVersionSortingStrategy.kt

```diff
@@ -0,0 +1,20 @@
+package com.amazon.spinnaker.keel.k8s.artifactSupplier
+
+import com.netflix.spinnaker.keel.api.artifacts.PublishedArtifact
+import com.netflix.spinnaker.keel.api.artifacts.SortingStrategy
+import com.netflix.spinnaker.keel.api.artifacts.TagVersionStrategy
+import com.netflix.spinnaker.keel.artifacts.TagComparator
+
+data class GitRepoVersionSortingStrategy(
+    val strategy: TagVersionStrategy,
+) : SortingStrategy {
+    override val type: String = "git-repo-versions"
+    private val tagComparator = TagComparator(strategy)
```

Comment:
do. we support all tagging strategies already available to docker except for custom regex?

Reply:
Correct. We just need to document